packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -154,6 +155,18 @@ export async function spawnPortableExperience(
     throw new Error(`Portable Scene: "${id}" is already running.`)
   }
 
+  let requiredPermissions: any = []
+  const sceneJsonFile = mappings.find((m) => m.file === 'scene.json')
```

Comment:
should this logic be available also for https://github.com/decentraland/kernel/pull/226/files#diff-64f6661a5927088d7905a509ad31c979d5fd25e760e4674cd5ace410aeb616ecL27 ?